.circleci/config.yml

```diff
@@ -1071,3 +1188,35 @@ workflows:
     jobs:
       - Demisto SDK Nightly:
           context: sdk_nightly_env
+
+  bucket_upload:
+    when: << pipeline.parameters.bucket_upload >>
+    triggers:
+      - schedule:
+          # should trigger every day at 12 AM UTC (3 AM Israel Time)
+          cron: "0 21 * * *"
```

Comment:
thats 9 PM UTC, i think we should make it later than that

Reply:
the build time is approx ~1.5 hours, so the latest time we can run it before nightly is 22